[23143] Avoid click propagation from active edit-fields

oliverguenther opened the pull request:
Avoid clicks within the form to bubble up to the row handler.
Otherwise, clicks within wp-edit-fields may cause the split / full view to open.

https://community.openproject.com/work_packages/23143/

TeatroIO commented:
I've prepared a stage to preview changes. <a href='http://oliverguenther-fix-split-screen-reload.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/' target='_blank'>Open stage</a> or <a href='http://oliverguenther-fix-split-screen-reload.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com//teatro.log' target='_blank'>view logs</a>.

oliverguenther committed:
Avoid click propagation from active edit-fields

Avoid clicks within the form to bubble up to the row handler.
Otherwise, clicks within wp-edit-fields may cause the split / full view to open.
